src/jit/flowgraph.cpp

```diff
@@ -13490,13 +13490,15 @@ bool Compiler::fgOptimizeSwitchBranches(BasicBlock* block)
         GenTree* switchVal = switchTree->gtOp.gtOp1;
         noway_assert(genActualTypeIsIntOrI(switchVal->TypeGet()));
 
+#ifndef LEGACY_BACKEND
```

Comment:
... and since none of the existing #ifndef's for JMPTABLE explain explicitly why it's switched out for LEGACY, I'll just assume that we don't need to explain it in the two spots where it was missing until this PR. Is that OK @briansull or do you want me to say something in the source? Thanks.